### PR TITLE
Adding support for Aeotec Water sensor to generic DTH

### PR DESCRIPTION
**Pull request by aclly**

### CODE REVIEW

**Comment by aclly** [comment]
@greens Can you review this when you get a chance?

File: devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy [hunk 1]
```diff
@@ -17,243 +17,256 @@
...
-	// Dome Leak Sensor sends WakeUpNotification every 12 hours. Please add zwaveinfo.mfr check when adding other sensors with different interval.
-	sendEvent(name: "checkInterval", value: (2 * 12 + 2) * 60 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+    // Dome Leak Sensor sends WakeUpNotification every 12 hours. Please add zwaveinfo.mfr check when adding other sensors with different interval.
+    sendEvent(name: "checkInterval", value: (2 * 12 + 2) * 60 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+}
+
+def configure() {
+    if (zwaveInfo.mfr.equals("0086")) {
+        def commands = []
+        commands << zwave.configurationV1.configurationSet(parameterNumber: 0x5E, scaledConfigurationValue: 1, size: 1).format()
```
Comment (greens) [inline devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy:70]: please describe what this parameter does in a comment

**Review by greens — CHANGES_REQUESTED** [review]
Can you revert back to tabs please?

**Comment by greens** [comment]
@aclly ping

**Comment by aclly** [comment]
We had a couple more things we needed to fix and are waiting on a firmware upgrade for the cell hub. Didn't want to push up the requested changes until it was ready to merge.

**Pull request closed without merging**